metabolinks/create_local_dbs.py

```diff
@@ -18,6 +18,35 @@
 
 """
 
+
+#Fetch and extract online data bases' .zip files
+def fetch_db(url, file_name):
+    import requests
+    from zipfile import ZipFile
+    # download the file contents in binary format
+    r = requests.get(url)
+    # open method to open a file on your system and write the contents
+    with open(file_name, "wb") as f:
+        f.write(r.content)
+
+    # opening the zip file in READ mode 
+    with ZipFile(file_name, 'r') as zip: 
+        # printing all the contents of the zip file 
+        zip.printdir() 
+  
+        # extracting all the files 
+        print('Extracting all the files now...') 
+        zip.extractall() 
+        print('Done!')
+
+print('Fetching hmbd...')
+fetch_db('http://www.hmdb.ca/system/downloads/current/hmdb_metabolites.zip', 'hmbd_db.zip')
+print('Fetching LIPIDMAPS...')
+fetch_db('http://www.lipidmaps.org/resources/downloads/LMSDFDownload12Dec17.zip', 'LIPIDMAPS_db.zip')
+
+
+#---------------------------------
+
 import xml.etree.ElementTree as ET
```

Comment:
Move these other imports also to the top of the script. They should all appear before the first def